ksw2_extd2_sse.c

```diff
@@ -4,14 +4,22 @@
 #include "ksw2.h"
 
 #ifdef __SSE2__
+#ifndef USE_SIMDE
 #include <emmintrin.h>
+#else
+#include <simde/x86/sse2.h>
+#endif
```

Comment:
Thanks for the update!

I think this part could like like this. And other parts are as well.
Because it's natural for me such as "if simde is used, include the simde header, otherwise include the default header.".

But let's wait @lh3 's response.

```
#ifdef USE_SIMDE
#include <simde/x86/sse2.h>
#else
#include <emmintrin.h>
#endif
```

Reply:
As there was not other response, I have changed C preprocessor conditionals to the above stated form.

Reply:
Thanks. It looks good to me.

Reply:
FYI, I personally always use `#define SIMDE_ENABLE_NATIVE_ALIASES` + `#include <simde/x86/sse2.h>` as the SIMDe library has fallback that get compiled away to the plain intrinsic calls when on x86 with the regular compilation flag (`-msse4.1`, etc).